gap/prop.gi

```diff
@@ -8,7 +8,47 @@
 #############################################################################
 ##
 
-#
+InstallMethod(IsMeetSemilatticeDigraph, "for a digraph",
+[IsDigraph],
+function(digraph)
+  local out_nbrs, i, j, intr;
+  if IsPartialOrderDigraph(digraph) then
+    out_nbrs := OutNeighbours(digraph);
+    for i in [1 .. Size(out_nbrs)] do
+      for j in [i + 1 .. Size(out_nbrs)] do
+        # Is there a better way of doing this? JDM
+        intr := Intersection(out_nbrs[i], out_nbrs[j]);
+        if not intr in out_nbrs then
```

Comment:
I'm pretty sure that this doesn't work, since `intr` is guaranteed to be sorted (it's a set, since it is produced by `Intersection`), however the entries of `out_nbrs` are not guaranteed to be sorted. So although `int` might equal `[1, 3, 5]` and `out[5]` might equal `[5, 3, 1]`, this line would return `false` even though I think you would want it to return `true`.